test/test_ops.py

```diff
@@ -506,39 +506,39 @@ def quote_strs(v):
 
                 if variant in method_or_inplace:
                     fn_template = '''
-                        def _fn(t0{c}{args_annot_kw}):
+                        def _fn(t0{c}{args}):
                             return t0.{alias_name}({args_kw})
                     '''
                     # remove the first input tensor
                     script = fn_template.format(
                         c=", " if len(args_kw[1:]) > 1 or len(args_annot_kw[1:]) >= 1 else "",
-                        args_annot_kw=", ".join(args_annot_kw[1:]),
+                        args=", ".join(args),
```

Comment:
Thanks for pointing this out, @Lezcano! Interestingly, we don't need this argument anymore since `args = ["to"]` and `args[1:]` is nothing but an empty list. I've removed this as an argument from method/inplace variants in the recent commit.

Reply:
Can you then update the comments accordingly?

Reply:
Since the `args=", ".join(args),` was removed but the comment still stays valid (as we do remove the first input tensor in `c` and `args_kw`), so I am not sure if there is any change in the comments needed. Please correct me if I'm missing anything here.